### DISCUSSION TITLE
Can't start container after updating config to Cloudflare

### DISCUSSION THREAD

**Discussion by JannikGHG** [qdm12/ddns-updater#872] [Q&A] [answered]
Hello guys,

after changing the config to Cloudflare because I moved the DNS function of my domain registrar Netcup to Cloudflare the container doesn't seem to start anymore.

I'm getting this error here:

2024-11-26T05:01:41-05:00 INFO reading JSON config from file /updater/data/config.json
2024-11-26T05:01:41-05:00 ERROR json: cannot unmarshal string into Go struct field .ttl of type uint32
2024-11-26T05:01:41-05:00 INFO Shutdown successful

Anyone is able to help?

**Comment by JannikGHG** [comment]
{
    "settings": [
        {
            "provider": "cloudflare",
            "domain": "*.removed.de,removed.de",
            "zone_identifier": "removed",
            "ttl": "1",
            "proxied": "true",
            "token": "removed"
        }
    ]
}

**Comment by qdm12** [comment, marked as answer]
The ttl you have set in your config.json is probably as a string, change it to a number

In example:

`"3600"` ➡️ `3600`

**Reply by JannikGHG** [reply]
ah I see, that's also what the error message literally says

also changed proxies from string to Boolean

up and running 👍🏻

(wow that answer was quick)
